### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by bgandon**
Hi,

These are the minor fixes [we discussed](https://groups.google.com/forum/#!topic/killbilling-users/wgW6PpLn-ec) two weeks ago on the mailing list.

Cheers

### CODE REVIEW

File: src/main/java/org/killbill/billing/plugin/avatax/api/AvaTaxInvoicePluginApi.java [hunk 1]
```diff
@@ -67,7 +66,7 @@ public AvaTaxInvoicePluginApi(final AvaTaxConfigurationHandler avaTaxConfigurati
 
     @Override
     public List<InvoiceItem> getAdditionalInvoiceItems(final Invoice invoice, final boolean dryRun, final Iterable<PluginProperty> properties, final CallContext context) {
-        final Collection<PluginProperty> pluginProperties = new ArrayList<PluginProperty>(ImmutableList.<PluginProperty>copyOf(properties));
+        final Collection<PluginProperty> pluginProperties = Lists.newArrayList(properties);
```
Comment (pierre) [inline src/main/java/org/killbill/billing/plugin/avatax/api/AvaTaxInvoicePluginApi.java]: Can you keep the explicit type argument (i.e. `Lists.<PluginProperty>newArrayList(properties)`)? Type inference is broken on old 1.6 jdk versions.

Comment (bgandon) [inline src/main/java/org/killbill/billing/plugin/avatax/api/AvaTaxInvoicePluginApi.java]: Of course yes!

**Comment by bgandon** [comment]
Btw, `mvn -P jdk16 clean test` builds but complains on uses of the deprecated `firstNonNull`.
I'll submit a trivial fix for this.

Plus, tests break on a missing `avatax.properties` file:
```
java.lang.IllegalArgumentException: resource avatax.properties not found.
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:145)
	at com.google.common.io.Resources.getResource(Resources.java:197)
	at org.killbill.billing.plugin.TestUtils.toString(TestUtils.java:104)
	at org.killbill.billing.plugin.TestUtils.loadProperties(TestUtils.java:77)
	at org.killbill.billing.plugin.avatax.AvaTaxRemoteTestBase.beforeMethod(AvaTaxRemoteTestBase.java:42)
```

**Comment by bgandon** [comment]
I tried a trivial fix, adding an empty `avatax.properties` in `src/test/resources` but it's definitely not sufficient.
Didn't you forget adding that file to your commits?

**Comment by pierre** [comment]
When running integration tests, you need AvaTax credentials in `src/test/resources/avatax.properties`. The default profile will only run unit (fast) tests (for which a sandbox is not required).

**Comment by pierre** [comment]
Thanks!

**Pull request merged**